packages/rspack/src/exports.ts

```diff
@@ -1,4 +1,6 @@
-const { version: rspackVersion, webpackVersion } = require("../package.json");
+import pkgJson from "@rspack/core/package.json";
```

Comment:
Relative path should be faster, no need to resolve

```suggestion
import pkgJson from "../package.json";
```

Reply:
<img width="400" alt="image" src="https://github.com/user-attachments/assets/2cacdd23-977e-4d82-bc40-47b02d8864a5">

I think "@rspack/core/package.json" is better for maintainability and "nodejs" resolve spec

Reply:
What do you mean by `"nodejs" resolve spec`, any document for this?